.gitignore

```diff
@@ -56,3 +56,5 @@ target/
 *~
 *.swp
 site/
+
+.idea
```

Comment:
That looks like something that is not specific to Ansible or this repository. Maybe you could use a [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)?